src/main/kotlin/commands/Nick.kt

```diff
@@ -57,16 +62,70 @@ class Nick(private val chattORE: ChattORE) : BaseCommand() {
         // Note: worth having a timeout to prevent people from changing too frequently.
         if (colors.isEmpty()) throw ChattoreException("No colors provided! Please provide 1 to 3 colors!")
         val rendered = if (colors.size == 1) {
-            val code = "<${colors.first().validateColor()}>"
-            val nickname = "${code}${player.username}<reset>"
+            val color = colors.first().validateColor();
+            val nickname = "<color:$color><username></color:$color>"
             chattORE.database.setNickname(player.uniqueId, nickname)
             nickname
         } else {
             if (colors.size > 3) throw ChattoreException("Too many colors!")
-            setNicknameGradient(player.uniqueId, player.username, *colors)
+            setNicknameGradient(player.uniqueId, *colors)
         }
         val response = chattORE.config[ChattORESpec.format.chattore].render(
-            "Your nickname has been set to $rendered".miniMessageDeserialize()
+            "Your nickname has been set to $rendered".render(mapOf(
+                "username" to Component.text(player.username)
+            ))
+        )
+        player.sendMessage(response)
+    }
+
+    @Subcommand("preset")
+    @CommandCompletion("@nickPresets")
+    fun pride(player: Player, preset: String) {
+        val format = chattORE.config[ChattORESpec.nicknamePresets][preset]
+            ?: throw ChattoreException("Unknown preset! Use /nick presets to see available presets.")
+        val rendered = format.render(mapOf("username" to Component.text(player.username)));
+        chattORE.database.setNickname(player.uniqueId, format);
+        val response = chattORE.config[ChattORESpec.format.chattore].render(
+            "Your nickname has been set to <message>".render(rendered)
+        )
+        player.sendMessage(response)
+    }
+
+    @Subcommand("presets")
+    @CommandCompletion("@username")
```

Comment:
This would also need an `@CommandPermission` tag. Probably `chattore.nick.presets`.

Reply:
Wouldn't it be better to share a permission? `/nick presets` has one purpose and that is to list the arguments to `/nick preset`. It doesn't make much sense to separate their permissions, as they are fundamentally part of the same functionality

Reply:
Yeah, that makes sense. Probably like `chattore.nick.preset` at that point.